VsVim.sln

```diff
@@ -71,17 +84,17 @@ Global
 		{11710F28-88D6-44DD-99DB-1F0AAA8CDAA0}.Release|Mixed Platforms.ActiveCfg = Release|Any CPU
 		{11710F28-88D6-44DD-99DB-1F0AAA8CDAA0}.Release|Mixed Platforms.Build.0 = Release|Any CPU
 		{11710F28-88D6-44DD-99DB-1F0AAA8CDAA0}.Release|x86.ActiveCfg = Release|Any CPU
-		{15DDED1B-E0CE-4128-9C63-A1D80171A08C}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{15DDED1B-E0CE-4128-9C63-A1D80171A08C}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{15DDED1B-E0CE-4128-9C63-A1D80171A08C}.Debug|Mixed Platforms.ActiveCfg = Debug|Any CPU
-		{15DDED1B-E0CE-4128-9C63-A1D80171A08C}.Debug|Mixed Platforms.Build.0 = Debug|Any CPU
-		{15DDED1B-E0CE-4128-9C63-A1D80171A08C}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{15DDED1B-E0CE-4128-9C63-A1D80171A08C}.Release|Any CPU.ActiveCfg = Debug|Any CPU
-		{15DDED1B-E0CE-4128-9C63-A1D80171A08C}.Release|Any CPU.Build.0 = Debug|Any CPU
-		{15DDED1B-E0CE-4128-9C63-A1D80171A08C}.Release|Mixed Platforms.ActiveCfg = Debug|Any CPU
-		{15DDED1B-E0CE-4128-9C63-A1D80171A08C}.Release|Mixed Platforms.Build.0 = Debug|Any CPU
-		{15DDED1B-E0CE-4128-9C63-A1D80171A08C}.Release|x86.ActiveCfg = Debug|Any CPU
-		{15DDED1B-E0CE-4128-9C63-A1D80171A08C}.Release|x86.Build.0 = Debug|Any CPU
+		{92121D37-D2E5-45AB-853C-99B0A85B7C8D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{92121D37-D2E5-45AB-853C-99B0A85B7C8D}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{92121D37-D2E5-45AB-853C-99B0A85B7C8D}.Debug|Mixed Platforms.ActiveCfg = Debug|Any CPU
+		{92121D37-D2E5-45AB-853C-99B0A85B7C8D}.Debug|Mixed Platforms.Build.0 = Debug|Any CPU
+		{92121D37-D2E5-45AB-853C-99B0A85B7C8D}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{92121D37-D2E5-45AB-853C-99B0A85B7C8D}.Release|Any CPU.ActiveCfg = Debug|Any CPU
+		{92121D37-D2E5-45AB-853C-99B0A85B7C8D}.Release|Any CPU.Build.0 = Debug|Any CPU
+		{92121D37-D2E5-45AB-853C-99B0A85B7C8D}.Release|Mixed Platforms.ActiveCfg = Debug|Any CPU
+		{92121D37-D2E5-45AB-853C-99B0A85B7C8D}.Release|Mixed Platforms.Build.0 = Debug|Any CPU
+		{92121D37-D2E5-45AB-853C-99B0A85B7C8D}.Release|x86.ActiveCfg = Debug|Any CPU
+		{92121D37-D2E5-45AB-853C-99B0A85B7C8D}.Release|x86.Build.0 = Debug|Any CPU

```

Comment:
I think this is related to my switch back and forth from the new SDK / CPS in VimCore. Unsure why VS didn't fix this the last time I did a change though.